README.md

```diff
@@ -17,7 +17,9 @@ Then install karton-dashboard from PyPi:
 ```shell
 $ pip install karton-dashboard
 
-$ karton-dashboard
+$ karton-dashboard run -h 127.0.0.1 -p 5000
 ```
 
+The `karton-dashboard` is just a wrapper on the `flask` program so you're free to provide it with arguments accepted by flask like `-h`, `--cert=path` and so on.
```

Comment:
```suggestion
The `karton-dashboard` is just a wrapper on the `flask` program, and it works with any arguments accepted by flask. For example `karton-dashboard --help`, or `karton-dashboard routes`. See [flask documentation](https://flask.palletsprojects.com/en/1.1.x/cli/) for more information.
```

(There is no such option as `-h`. And no option will work without karton config. But maybe no point in being that detailed)